ktor-server/ktor-server-netty/jvm/src/io/ktor/server/netty/NettyApplicationEngine.kt

```diff
@@ -67,6 +67,11 @@ public class NettyApplicationEngine(
          * User-provided function to configure Netty's [HttpServerCodec]
          */
         public var httpServerCodec: () -> HttpServerCodec = ::HttpServerCodec
+
+        /*
```

Comment:
kdoc format: `/**` at the first line and `*/` at the last

Reply:
Fixed